src/test/java/org/springframework/integration/kafka/outbound/KafkaProducerMessageHandlerTests.java

```diff
@@ -195,8 +196,6 @@ public void testOutbound() throws Exception {
 		assertThat(record).has(key(2));
 		assertThat(record).has(partition(1));
 		assertThat(record.value()).isNull();
-
-		producerFactory.destroy();
```

Comment:
Why have you removed this one? `Cut-and-paste` artefact? 😉

Reply:
Darn, this is stupid!